source/trackers.json

```
@@ -1157,8 +1187,14 @@
         "adjust.net.in": "adjust",
         "adjust.world": "adjust",
         "apptrace.com": "adjust",
+        "admixer.com": "admixer",
+        "admixer.net": "admixer",
         "adobe.io": "adobe_developer",
         "adobedc.net": "adobe_experience_league",
+        "adprofex.com" : "adprofex",
+        "ads2.bid" : "adprofex",
+        "cz.affilbox.cz": "affilbox",
+        "affilbox.com": "affilbox",
```

Порядок не по возрастанию, поддомен вроде можно убрать
```suggestion
        "affilbox.com": "affilbox",
        "affilbox.cz": "affilbox",
```